homeassistant/components/yolink/__init__.py

```diff
@@ -0,0 +1,122 @@
+"""The yolink integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+
+import async_timeout
+import voluptuous as vol
+from yolink_client.const import OAUTH2_AUTHORIZE, OAUTH2_TOKEN
+from yolink_client.yolink_client import YoLinkClient
+from yolink_client.yolink_mqtt_client import HomeEventMqttSub, MqttClient
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_CLIENT_ID, CONF_CLIENT_SECRET
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import (
+    aiohttp_client,
+    config_entry_oauth2_flow,
+    config_validation as cv,
+)
+from homeassistant.helpers.typing import ConfigType
+
+from . import api, config_flow
+from .const import DOMAIN, HOME_ID, HOME_SUBSCRIPTION
+
+SCAN_INTERVAL = timedelta(minutes=5)
+
+_LOGGER = logging.getLogger(__name__)
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN: vol.Schema(
+            {
+                vol.Required(CONF_CLIENT_ID): cv.string,
+                vol.Required(CONF_CLIENT_SECRET): cv.string,
+            }
+        )
+    },
+    extra=vol.ALLOW_EXTRA,
+)
+
+PLATFORMS = ["sensor"]
```

Comment:
```suggestion
PLATFORMS = [Platform.SENSOR]
```

and add `from homeassistant.const import Platform`